docs/README_RECON.md

```diff
@@ -138,30 +212,79 @@ class ColumnMapping:
 
 ### transformations
 
-```python
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
+
 @dataclass
 class Transformation:
     column_name: str
     source: str
     target: str | None = None
-```
+
+</pre>
+</td>
+<td>
+<pre lang="json">
+"transformations":[
+    {
+      "column_name": "product_price",
+      "source": "CAST(product_price AS DECIMAL(18, 2))",
+      "target": "CAST(p_price AS DECIMAL(18, 2))"
+    }
+]
+</pre>
+</td>
+</tr>
+</table>
+
 
 | field_name  | data_type | description                                                | required/optional | example_value                    |
 |-------------|-----------|------------------------------------------------------------|-------------------|----------------------------------|
 | column_name | string    | the column name on which the transformation to be applied  | required          | "s_address"                      |
 | source      | string    | the transformation sql expr to be applied on source column | required          | "trim(s_address)" or "s_address" |
 | target      | string    | the transformation sql expr to be applied on source column | required          | "trim(s_address)" or "s_address" |
 
-### thresholds
+## thresholds
+
+<table>
+<tr>
+<th>python</th>
+<th>json</th>
+</tr>
+<tr>
+<td>
+<pre lang="python">
 
-```python
 @dataclass
 class Thresholds:
     column_name: str
     lower_bound: str
     upper_bound: str
     type: str
-```
+
+</pre>
+</td>
+<td>
+<pre lang="json">
+"thresholds":[
+  {
+    "column_name": "p_price",
```

Comment:
Please update all the actual values with <TO-BE-FILLED> elements.

Reply:
Fixed